pavlo976/BuildMinder-App
# Feature/add employee form

### What's changed?
AddEmployeeForm component was created
### How to test it?
Import to Employee page AddEmployeeForm and on the website try to send wrong data to check validation. Also try to send correct data and check if the behavior of the form after correct data submission is correct
### What's missing?
Everything is up to date
